port-chooser/build.gradle.kts

```diff
@@ -0,0 +1,79 @@
+plugins {
+    id("org.terracotta.java-conventions")
+    id("org.terracotta.utilities.plugins.plantuml")
+}
+
+val logbackRangeVersion: String by project
+
+description = "Terracotta Utilities Port Chooser"
+
+dependencies {
+    api(project(":terracotta-utilities-tools"))
+    testImplementation(project(":terracotta-utilities-test-tools"))
+    testImplementation("ch.qos.logback:logback-classic:${logbackRangeVersion}")
+}
+
+testing {
+    suites {
+        named<JvmTestSuite>(JvmTestSuitePlugin.DEFAULT_TEST_SUITE_NAME) {
+            targets {
+
+                val testIPv4_base by creating {
+                    // Run test using IPv4 preference
+                    testTask {
+                        filter {
+                            setIncludePatterns("org.terracotta.utilities.test.net.*Test")
+                            // Exclude single test run below
+                            setExcludePatterns("org.terracotta.utilities.test.net.PortManagerTest")
+                        }
+                        systemProperty("java.net.preferIPv4Stack", "true")
+                        systemProperty("org.terracotta.disablePortReleaseCheck", "false")
+                        environment(
```

Comment:
There is an overload of the `environment` method that can take the list of mappings directly:
```kotlin
environment("DISABLE_PORT_RELEASE_CHECK" to "")
```

Reply:
Changed.